buildconfig/stubs/pygame/constants.pyi

```diff
@@ -560,5 +560,3 @@ WINDOWRESTORED: int
 WINDOWSHOWN: int
 WINDOWSIZECHANGED: int
 WINDOWTAKEFOCUS: int
-
-__all__: List[str]
```

Comment:
You have to edit this [file](https://github.com/pygame/pygame/blob/main/buildconfig/stubs/gen_stubs.py) to remove that line

Reply:
Oh right, good catch. Thanks!